### PR TITLE
Added emojis as icons for the emoji groups

### PR DESCRIPTION
**Pull request by ZKLaurin**
Changed white Smileys to Emojis representing the group.
image

### CODE REVIEW

File: src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue [hunk 1]
```diff
@@ -90,6 +98,20 @@ const favorites = useConfig<Set<string>>("ui.emote_menu.favorites");
 // const usage = useConfig<Map<string, number>>("ui.emote_menu.usage");
 // const shouldShowUsage = useConfig<boolean>("ui.emote_menu.most_used");
 
+//Emote icons for groups
```
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue:101]: nit: add whitespace at comment start

```suggestion
// Emote icons for groups
```

File: src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue [hunk 2]
```diff
@@ -29,6 +29,13 @@
 					>
 						<div class="set-sidebar-icon">
 							<img v-if="es.owner && es.owner.avatar_url" :src="es.owner.avatar_url" />
+							<div v-else-if="es.provider === 'EMOJI'" class="emoji-group">
+								<SingleEmoji
+									:id="emojiGroupIcons[emojiCategories.indexOf(es.name)]"
+									:alt="es.name"
+									:style="{ width: `100%`, height: `100%` }"
+								/>
+							</div>
```
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue:38]: nit: it makes more sense to invert this condition; while it's unlikely `owner` would ever be defined on an emoji set, if it ever was then this means the special emoji case wouldn't render.

```suggestion
							<div v-if="es.provider === 'EMOJI'" class="emoji-group">
								<SingleEmoji
									:id="emojiGroupIcons[emojiCategories.indexOf(es.name)]"
									:alt="es.name"
									:style="{ width: `100%`, height: `100%` }"
								/>
							</div>
							<img v-else-if="es.owner && es.owner.avatar_url" :src="es.owner.avatar_url" />
```

File: src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue [hunk 3]
```diff
@@ -29,6 +29,13 @@
 					>
 						<div class="set-sidebar-icon">
 							<img v-if="es.owner && es.owner.avatar_url" :src="es.owner.avatar_url" />
+							<div v-else-if="es.provider === 'EMOJI'" class="emoji-group">
+								<SingleEmoji
+									:id="emojiGroupIcons[emojiCategories.indexOf(es.name)]"
+									:alt="es.name"
+									:style="{ width: `100%`, height: `100%` }"
```
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue:36]: The inline style may not be necessary as you already define a class in the outer div. Target this element within it

Comment (ZKLaurin) [reply src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue]: I removed this styling in commit 1dda7d0

File: src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue (hunk 2 above)
Comment (ZKLaurin) [reply src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue]: Changed the order in 1dda7d0 .
However, I don't know if this is the right thing to do, because you have to specify a link in the owner that you probably want to display. It should be fine either way for this special scenario and I trust you have more experience for this stuff.

File: src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue (hunk 1 above)
Comment (ZKLaurin) [reply src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue]: Added whitespace in 1dda7d0

**Review by AnatoleAM — COMMENTED** [review]
Nice, maybe though the categories could be sorted according to standard?

Which is, I believe:
- Smileys & Emoticon
- People & Body
- Animals & Nature
- Food & Drink
- Activities
- Travels & Places
- Objects
- Symbols
- Flags

**Comment by ZKLaurin** [comment]
I added a condition in the sorting function to sort after the `emojiCategories` Array.
This was the only place i could really find to sort this, since the EmoteSet is a Record Type and is always sorted alphabetically.
Hopefully this approach is still fine with you :)

**Comment by ZKLaurin** [comment]
I messed up the rebase, but nothing changed except the changelog...

File: src/site/twitch.tv/modules/hidden-elements/hiddenElements.ts
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/hidden-elements/hiddenElements.ts:1]: Why is this here?

**Comment by ZKLaurin** [comment]
@AnatoleAM Sorry for the previous push, i reset it and merged it with the master branch.
